server/index.js

```diff
@@ -54,6 +43,30 @@ app.post('/api-reviews', (req, res) => {
     }
   });
 });
+
+app.get('/api-reviews/:productid', (req, res) => {
+  db.getReviewsByProductId(req.params.productid, (err, results) => {
+    if (err) {
+      throw err;
+    } else {
+      res.status(200);
+      res.end(JSON.stringify(results));
+    }
+  });
+});
+
+// update
+app.put('/api-increment', (req, res) => {
```

Comment:
I think robin said that put requests should have the id of the entry you are modifying in the url rather than the request body.

Reply:
This was originally there and if I modify it, it's going to affect the frontend.